resource_control: RU info in explain analyze (#14234)
pingcap/docs: tidb-resource-control.md

> - In addition to queries executed by users, RU can be consumed by background tasks, such as automatic statistics collection.
> - The preceding table lists only the resources involved in RU calculation for TiDB Self-Hosted clusters, excluding the network and storage consumption. For TiDB Serverless RUs, see [TiDB Serverless Pricing Details](https://www.pingcap.com/tidb-cloud-serverless-pricing-details/).

> ### Predict RU consumption of SQL statements

Reviewer: ```suggestion
## Estimate RU consumption of SQL statements
```

> [`explain analyze`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) shows the actual RU produced by one particular SQL statement. Please be aware the RU consumption could vary if the same SQL statement repeats, as the amount of RU is affected by cache layer (for example, [coprocessor cache](/coprocessor-cache.md)). Even though this value doesn't reflect the exact RU produced by each execution, we can still come with an estimation based on this value. 

Reviewer: ```suggestion
[`explain analyze`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) shows the actual RU produced by one particular SQL statement. Note that the RU consumption might vary if the same SQL statement repeats, as the amount of RU is affected by the cache layer (for example, [coprocessor cache](/coprocessor-cache.md)). Even though this value does not reflect the exact RU produced by each execution, you can still come with an estimation based on this value. 
```

> [`explain analyze`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) shows the actual RU produced by one particular SQL statement. Note that the RU consumption might vary if the same SQL statement repeats, as the amount of RU is affected by the cache layer (for example, [coprocessor cache](/coprocessor-cache.md)). Even though this value does not reflect the exact RU produced by each execution, you can still come with an estimate the RU consumption based on this value. 

Reviewer: ```suggestion
You can execute [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) to get the amount of RU consumed during SQL execution. Note that the amount of RU is affected by the cache ( for example, [coprocessor cache](/coprocessor-cache.md)). When executing the same SQL multiple times, the amount of RUs consumed by each execution might be different. The RU value does not represent the exact value for each execution, but can be used as a reference for estimation.
```

> You can execute [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) to get the amount of RU consumed during SQL execution. Note that the amount of RU is affected by the cache ( for example, [coprocessor cache](/coprocessor-cache.md)). When executing the same SQL multiple times, the amount of RUs consumed by each execution might be different. The RU value does not represent the exact value for each execution, but can be used as a reference for estimation.

Reviewer 2: ```suggestion
You can use the [`EXPLAIN ANALYZE`](/sql-statements/sql-statement-explain-analyze.md#ru-request-unit-consumption) statement to get the amount of RUs consumed during SQL execution. Note that the amount of RUs is affected by the cache (for example, [coprocessor cache](/coprocessor-cache.md)). When the same SQL is executed multiple times, the amount of RUs consumed by each execution might be different. The RU value does not represent the exact value for each execution, but can be used as a reference for estimation.
```
